qt/scientific_interfaces/Indirect/Iqt.cpp

```diff
@@ -340,8 +353,8 @@ bool Iqt::validate() {
   uiv.checkDataSelectorIsValid("Sample", m_uiForm.dsInput);
   uiv.checkDataSelectorIsValid("Resolution", m_uiForm.dsResolution);
 
-  const auto eLow = m_dblManager->value(m_properties["ELow"]);
-  const auto eHigh = m_dblManager->value(m_properties["EHigh"]);
+  auto const eLow = m_dblManager->value(m_properties["ELow"]);
```

Comment:
`const auto` seems to be used more in the Mantid codebase, but `auto const` seems to be used everywhere in this file - I'm torn on whether it's worth changing it

Reply:
I prefer to use east `const` because you can reliable use it in any situation, whereas there are cases where west `const` can't be used, and you must use east `const` instead. For instance when you have a const member function of a class but the return type is not const - then you have no choice but to use east `const`.

Reply:
I see your point. My only concern is that most of the codebase uses west const and Indirect classes seem to be one of the few that stray from that

Reply:
I understand your concern. Would you be ok with me just taking this advice forward for the future - changing these east const's in the Indirect interfaces would take a lot of time?

Reply:
Of course, that is fine with me